tests/apm/monitoring-commandFailed-003.phpt

```diff
@@ -48,7 +48,7 @@ try {
 --EXPECTF--
 started: findAndModify
 failed: findAndModify
-object(stdClass)#%d (%d) {
+object(stdClass)#%d (%d) {%A
```

Comment:
4.0.0-rc5 replica set reports "operationTime" as the first field in the command response.